omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/RegisterPatientFragmentController.java

```diff
@@ -311,7 +311,7 @@ private void buildGroupObs(ConceptService conceptService, List<Obs> obsToCreate,
                 for (ObsGroupItem obsGroupItem : obsGroupItems) {
                     buildObs(conceptService, groupObsToCreate, obsGroupItem.getObsConcept(), obsGroupItem.getObsValues());
                 }
-                if (groupObsToCreate.size() > 0) {
+				if (!groupObsToCreate.isEmpty()) {
```

Comment:
```suggestion
                if (!groupObsToCreate.isEmpty()) {
```
Same as above.

Reply:
Ok, will make updates to match the initial format.